contracts/Marketplace.sol

```diff
@@ -168,9 +168,10 @@ contract Marketplace is Proofs, StateRetrieval {
 
     _removeFromMySlots(slot.host, slotId);
 
+    uint256 slotIndex = slot.slotIndex;
```

Comment:
Interesting! 😅

Reply:
I'm not sure how the nim-codex contracts tests were passing without this 🤔 Perhaps nim-codex was on an older commit without this change 🤷